src/views/v2/event-status/single/canceled-status.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Status for a Canceled Event.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/events-status/single/canceled-status.php
```

Comment:
if we are following the folder structure we've used for the other views, this should live in: `tribe/events/v2/events-status/single/event-statuses/canceled-status.php`